x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/api.ts

```diff
@@ -24,7 +25,8 @@ import {
   UpdateAlertStatusProps,
   CasesFromAlertsResponse,
 } from './types';
-import { isolateHost } from '../../../../common/lib/host_isolation';
+import { resolvePathVariables } from '../../../../management/pages/trusted_apps/service/utils';
```

Comment:
Ok for now, but I think either @ashokaditya or @dasansol92 also had the need to use this utility in Event Logs. We should promote it to `security_solution/public/common/utils/*` so that its clearer that it is reusable across other areas of security solution.